src/utils.ts

```diff
@@ -440,5 +440,5 @@ export const focusNearestParent = (element: HTMLInputElement) => {
 };
 
 export const focusContainer = () => {
-  document.querySelector<HTMLElement>(".excalidraw")?.focus();
+  document.querySelector<HTMLElement>(".excalidraw-container")?.focus();
 };
```

Comment:
Now that we have this, do you think there's any benefit to keeping the class method `focusContainer` (and the added complexity of passing it down to components) or can we use this everywhere? IMO there shouldn't ever be any React race conditions that will make this fail, and if there are it won't be the end of the world.

Reply:
actually, just realized this won't work with multi-components :/

Reply:
Yeah i have passed the focus container reference for this case but there still are issues since library is shared by all excal containers so will fix that in separate pr and also for error dialog case we should always focus back to the element which triggered it but thats not possible incase of library error dialog coz the library pop up is closed so I will fix that in separate PR along with removing redundant code of restoring focus for dialog